src/allotropy/allotrope/schema_mappers/adm/pcr/BENCHLING/_2023/_09/dpcr.py

```diff
@@ -234,3 +291,61 @@ def _get_calculated_data_aggregate_document(
                 for calculated_data_item in calculated_data_items
             ]
         )
+
+    CubeClass = TypeVar("CubeClass")
```

Comment:
Need to move this outside of the class definition (above `class Mapper`) after that, LGTM!

Reply:
👍